model/internal/pdata/generated_log.go

```diff
@@ -640,6 +640,8 @@ func (ms LogRecord) SetSeverityNumber(v SeverityNumber) {
 }
 
 // Name returns the name associated with this LogRecord.
+//
+// Deprecated: [v0.48.0] it was removed from the data model.
```

Comment:
Is there a recommendation for current users of this? Would just removing this call result in the same data quality, or should this `name` be added as an attribute?

Reply:
Is the same information supposed to be conveyed in a different way?

Reply:
No. The field is deprecated because there is no clear understanding on what the field is supposed to carry. Hence, no replacement either. It is unconditionally removed without attempting to support it in any graceful way. There is an ongoing discussion about whether we can have a proper definition of what "Name" should be with precisely defined semantics. If that happens we may add the field, but it will be considered a completely new field, without any graceful transition required.

After keeping this deprecation notice in the codebase we should just delete the field soon.

Reply:
Great, thanks for the clarification.